### DISCUSSION TITLE
Uncaught TypeError: Failed to resolve module specifier “Vue”. Relative references must start with either “/”, “./”, or “…/”.

### DISCUSSION THREAD

**Discussion by capoia** [nuxt/nuxt#16874] [Questions] [answered]
Hello, I'm trying to deploy a nuxt project on netlify, but it's returning this error, it only happens when it goes to the server (test on netlify and digital ocean apps).
The problem is that you don't have a stacktrace to know where this error is coming from, any ideas on how to debug this to solve it?
(I can't simulate on local computer, here I use npm run build and npm run start and everything works)
(no error is generated in the build either, as you can see in the attachment.)

example site: https://rainbow-muffin-dbdc67.netlify.app/
[logs.docx](https://github.com/nuxt/framework/files/9566236/logs.docx)

**Comment by danielroe** [comment, marked as answer]
Just a guess, but you are probably developing on a case-insensitive Mac. But your deployment target is Linux and it cares about case. So find the import from 'Vue' in your code and change it to 'vue'.
